picard/_core_picard.py

```diff
@@ -8,11 +8,13 @@
 from scipy.linalg import expm
 
 from .densities import Tanh
+from ._tools import fuse_mask
 
 
 def core_picard(X, density=Tanh(), ortho=False, extended=False, m=7,
                 max_iter=1000, tol=1e-7, lambda_min=0.01, ls_tries=10,
-                verbose=False, covariance=None):
+                verbose=False, covariance=None, ll_reject=None,
+                reject_every=None):
```

Comment:
Not sure how to set a default value for `ll_reject` if the user wants to use rejection